src/test/resources/aggregatedSourceChangeObject.json

```diff
@@ -0,0 +1 @@
+[{ "_id" : "fb6ef12d-25fb-4d77-b9fd-5fktsrefe66de47" , "aggregatedObject" : "{\"ongoing\":\"false\",\"TemplateName\":\"TEST_EXECUTION_1\",\"id\":\"e1e93f13-7c3c-3f17-975f-ebf0cd6ff1c2\",\"time\":1234567890,\"type\":\"EiffelActivityFinishedEvent\",\"version\":\"1.0.0\",\"outcome\":{\"conclusion\":\"TIMED_OUT\",\"description\":\"Compilation timed out.\"}}"}]
```

Comment:
New coming EI versions has solved the Escape character issues in Aggregated (JSON)Objects, it will soon be merge to EI main repository. When that has been done, then rebase with latest EI and remove the escaped characters(plus extra quotations) from Aggregated JSON object.

Reply:
If you rebase, or how it works in GitHub, and get the latest EI, you will get EI created Aggregated Objects without Escape Character("\""). After rebase you can remove all escape character in this Json File. and indent your json file. If using Eclipse, mark whole line with Ctrl-A, and JSON indent with key combination Ctrl+Shift-F.

Reply:
Check AggregatedDocument.json file how it should look like.